.changes/v3.10.0/1079-notes.md

```diff
@@ -0,0 +1 @@
+* Bump `terraform-plugin-sdk` to v2.27.0 [GH-1079]
```

Comment:
My only question if it's not too late to do the bump now given how close we are to release?

CC: @dataclouder @adezxc @Didainius

Reply:
Never mind, I just realized it's the security-required bump, so let's bite the bullet and do it sooner.